DOCUMENTATION.md

```diff
@@ -0,0 +1,155 @@
+Other files | `--filesystem=[PATH]`, `--filesystem=[PATH]:ro` and `--nofilesystem=[PATH]` | Input | Permit (`--filesystem=[PATH]`) or prohibit (`--nofilesystem=[PATH]`) read-write access to the application to the directory you desire. <br /> <br /> For example, you would put `~/games` if you want read-write access to `~/games`. If you want read-only (`ro`) access to `~/games`, then it will be `~/games:ro`.
+
+#### Persistent
+
+List of the homedir-relative paths created in the sandbox.
+
+Name | `flatpak-override` equivalent | Type | Description
+--- | --- | --- | ---
+Files | `--persist=[PATH]` | Input | Permit only to the application to have access to the targeted directory while restricting other applications from accessing it. <br /> <br /> Starting from `$HOME`, the targeted directory will be remapped to the Flatpak program's directory (`~/.var/app/$FLATPAK_APP_ID/[PATH]`) if it has no write access to the targeted directory. <br /> <br /> For example, persisting `.mozilla` will map `~/.mozilla` to `~/.var/app/org.mozilla.Firefox/.mozilla` if the Firefox Flatpak has no write access to `~/.mozilla`.
+
+#### Environment
+
+List of variables exported to the application.
+
+Name | `flatpak-override` equivalent | Type | Description
+--- | --- | --- | ---
+Variables | `--env=[VAR]=[VALUE]` | Input | Set an environment variable in the application.
+
+#### System Bus
+
+List of well-known names on the system bus.
+
+Name | `flatpak-override` equivalent | Type | Description
+--- | --- | --- | ---
+Talks | `--system-talk-name=NAME` | Input | Allow the application to talk to the well known name NAME on the session bus. If NAME ends with .*, it allows the application to talk to all matching names. This updates the [Session Bus Policy] group in the metadata. This option can be used multiple times. 
+Owns | `--system-own-name=NAME` | Input | Allow the application to own the well known name NAME on the session bus. If NAME ends with .*, it allows the application to own all matching names. This updates the [Session Bus Policy] group in the metadata. This option can be used multiple times. 
+
+#### Session Bus
+
+List of well-known names on the session bus.
+
+Name | `flatpak-override` equivalent | Type | Description
+--- | --- | --- | ---
+Talks | `--talk-name=NAME` | Input | Allow the application to talk to the well known name NAME on the session bus. If NAME ends with .*, it allows the application to talk to all matching names. This updates the [Session Bus Policy] group in the metadata. This option can be used multiple times. 
+Owns | `--own-name=NAME` | Input | Allow the application to own the well known name NAME on the session bus. If NAME ends with .*, it allows the application to own all matching names. This updates the [Session Bus Policy] group in the metadata. This option can be used multiple times. 
```

Comment:
If  the user knows what Dbus is and what the session and system bus do, this works... but ideally, we could simplify this a bit by using friendlier terms like "system services the application can talk to, and that it can offer", for "talk" we can use the org,freedesktop.Notifications as an example, saying that it means it can send notifications.

Reply:
https://github.com/tchx84/Flatseal/pull/136/commits/a9ccbc9ded38e2a72c4213195b4c650a1268d7b3